### PR TITLE
#[563] Edit the descriptions of some surveys

### PR DESCRIPTION
**Pull request by jhflorey**
This PR addresses the issue https://github.com/episphere/connectApp/issues/563#issuecomment-1640419504

- Updated description of modules['Biospecimen Survey'] and modules['Clinical Biospecimen Survey'] in myToDoList.js`

### CODE REVIEW

**Review by JoeArmani — APPROVED** [review]
looks good

Will we be adding back the commente out code soon, or should it be removed?

File: js/pages/myToDoList.js [hunk 1]
```diff
@@ -707,26 +707,26 @@ const setModuleAttributes = (data, modules, collections) => {
...
 
-    if(data['331584571']?.['266600170']?.['840048338']) {
+    // if(data['331584571']?.['266600170']?.['840048338']) {
         modules['Biospecimen Survey'].enabled = true;
         modules['Covid-19'].enabled = true;
-    }
+    // }
 
-    if(collections && collections.filter(collection => collection['650516960'] === 664882224).length > 0) {
+    // if(collections && collections.filter(collection => collection['650516960'] === 664882224).length > 0) {
```
Comment (anthonypetersen) [inline js/pages/myToDoList.js:726]: why is this being commented out?

**Review by anthonypetersen — CHANGES_REQUESTED** [review]

File: js/pages/myToDoList.js [hunk 2]
```diff
@@ -707,26 +707,26 @@ const setModuleAttributes = (data, modules, collections) => {
...
-    modules['Clinical Biospecimen Survey'].description = 'Questions about recent actions, like when you last ate and when you went to sleep and woke up on the day you donated samples, and your history of COVID-19. ';
+    modules['Clinical Biospecimen Survey'].description = 'Questions about recent actions, like when you last ate and when you went to sleep and woke up on the day you donated samples.';
     modules['Clinical Biospecimen Survey'].estimatedTime = '5 minutes';
 
     modules['Menstrual Cycle'].header = 'Menstrual Cycle Survey';
     modules['Menstrual Cycle'].description = 'Questions about the date of your first menstrual period after you donated samples for Connect. ';
     modules['Menstrual Cycle'].estimatedTime = '5 minutes';
 
-    if(data['331584571']?.['266600170']?.['840048338']) {
+    // if(data['331584571']?.['266600170']?.['840048338']) {
```
[the remark this answers is not in the document]
Comment (jhflorey) [reply js/pages/myToDoList.js]: Just for testing, my mistake.

File: js/pages/myToDoList.js (hunk 1 above)
Comment (jhflorey) [reply js/pages/myToDoList.js]: Just for testing, my mistake.

**Review by anthonypetersen — APPROVED** [review]
looks good

**Pull request merged**